modules/custom/az_migration/src/Plugin/migrate/process/ParagraphsColumnImageFieldMerge.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Drupal\az_migration\Plugin\migrate\process;
+
+use Drupal\migrate\MigrateExecutableInterface;
+use Drupal\migrate\ProcessPluginBase;
+use Drupal\migrate\Row;
+use Drupal\media\Entity\Media;
+
+/**
+ * Process Plugin to field merge for Column Image paragraphs.
+ *
+ * @MigrateProcessPlugin(
+ *   id = "paragraphs_column_image_field_merge"
+ * )
+ */
+class ParagraphsColumnImageFieldMerge extends ProcessPluginBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function transform($value, MigrateExecutableInterface $migrate_executable, Row $row, $destination_property) {
+    // Merging the data into one field.
+    $value['value'] = '';
+    // Getting the media into text.
+    if (isset($value[0])) {
+      $media = Media::load($value[0]);
+      if (!empty($media)) {
+        $value['value'] .= '<drupal-media data-align="center" data-entity-type="media" data-entity-uuid="' . $media->get('uuid')->value . '" data-view-mode="az_large"></drupal-media>';
```

Comment:
`data-align` should be none (or left if none isn't an option)

Reply:
@danahertzberg  Will add that in..thanks

Reply:
Done